rotation/roots.go

```diff
@@ -103,9 +105,12 @@ func RotateRootCertificates(ctx context.Context, storage nodeenrollment.Storage,
 		{
 			now := time.Now()
 			template := &x509.Certificate{
-				AuthorityKeyId:        newRoot.PublicKeyPkix,
-				SubjectKeyId:          newRoot.PublicKeyPkix,
-				DNSNames:              []string{nodeenrollment.CommonDnsName},
+				AuthorityKeyId: newRoot.PublicKeyPkix,
+				SubjectKeyId:   newRoot.PublicKeyPkix,
+				Subject: pkix.Name{
```

Comment:
should we have a test that asserts the cert Subject and DNSNames are
appropriately set?

Reply:
I think I skipped that because it really doesn't matter all that much (we don't use the key ID here, instead we calculate it from the cert's embedded key each time, but I can add for sure.